Description: Update dependencies to match packaged versions
Author: Arnaud Ferraris <aferraris@debian.org>
Forwarded: not-needed
Last-Update: 2024-10-11
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -91,6 +91,6 @@
 version = "3"
 
 [build-dependencies.system-deps]
-version = "6"
+version = "7"
 
 [features]
